### PR TITLE
Added FFXIV: Shadowbringers pre-order item "Baby Gremlin" 

### PR DESCRIPTION
**Pull request by Crakila**
As per title. 

The item was distributed to people as of Friday, March 1st 2019 who have pre-ordered Final Fantasy XIV: Shadowbringers. 

Please feel free to double-check as I just search for anything that was `sb` related and added an `h` in the middle 😛 

Anything that is `sb` is `shb` and anything that is `SB` is `ShB` (assuming that we are using the `ShB` code for the expansion since Square-Enix hasn't formally decided on one) 

Please double-check and do whatever Java magic that you guys do. 😂

### CODE REVIEW

File: src/main/resources/db/changelog/db.changelog-master.yaml [hunk 1]
```diff
@@ -280,6 +286,12 @@ databaseChangeLog:
                   defaultValueNumeric: 0
                   constraints:
                     nullable: false
+              - column:
```
Comment (matthewhillier) [inline src/main/resources/db/changelog/db.changelog-master.yaml:289]: Please create a new changeset entry for your new column. This process works by only applying change sets to the database that haven't already been run, so this won't be applied on an existing census DB. 😄

**Comment by matthewhillier** [comment]
Hey @fahy - overall looks good! Just one change to make on the database changelog file as noted.

Also, please pull from master and rebase, as github is showing that your merge is going to undo some of the more recent commits to master.

Thanks!! 😄

**Comment by Pricetx** [comment]
Once the mentioned issues are sorted we should be able to begin testing, as it will have given players a few days to redeem theirs.

**Comment by Crakila** [comment]
Closing this. #47 is the proper PR :)

**Pull request closed without merging**